docs/bitbox02.md

```diff
@@ -0,0 +1,49 @@
+# BitBox02
+
+The BitBox02 is supported by HWI.
+
+Current implemented commands are:
+
+* `signtx`
+* `getxpub`
+* `displayaddress`
+* `setup`
+* `wipe`
+* `restore`
+* `backup`
+* `togglepassphrase`
+
+Multisig (P2WSH only) is supported by the BitBox02, but is not ingerated into HWI yet. Coming
+soon^{tm}.
```

Comment:
What is the blocker here?

Reply:
- Passing the full descriptor like in https://github.com/bitcoin-core/HWI/pull/353 so the setup can be registered
- I figured smaller PRs are better. Multisig will add quite a bit of code, but this PR is already big enough to look over